pyte/screens.py

```diff
@@ -239,7 +239,12 @@ def render(line):
                     continue
                 char = line[x].data
```

Comment:
I'm not sure if an empty string is possible here unless assigned explicitly. Do you have an example of a terminal app triggering this?